lib/MailQueueHandler.php

```diff
@@ -270,9 +271,15 @@ public function sendEmailToUser($userName, $email, $lang, $timezone, $maxTime) {
 		$message->setSubject((string) $l->t('Activity notification'));
 		$message->setPlainBody($emailText);
 		$message->setFrom([$this->getSenderData('email') => $this->getSenderData('name')]);
-		$this->mailer->send($message);
+
+		try {
+			$this->mailer->send($message);
+		} catch (\Exception $e) {
```

Comment:
This is always really bad 🙈 Then an exception is simply dropped - at least having it in debug mode visible would make this traceable.